includes/Traits/Types.php

```diff
@@ -151,7 +151,7 @@ public function get_allowed_image_mime_types() {
 		 */
 		$image_mime_type = apply_filters( 'web_stories_allowed_image_mime_types', $image_mime_type, $allowed_mime_type );
 
-		return array_intersect( $allowed_mime_type, $image_mime_type );
+		return array_values( array_intersect( $allowed_mime_type, $image_mime_type ) );
```

Comment:
This brings this inline this line. 

https://github.com/google/web-stories-wp/blob/82088c76714904e49b9a289f45184b194316fe2d/includes/Traits/Types.php#L120